docs/add-non-um-users.html

```diff
@@ -0,0 +1,9 @@
+---
+layout: default
+title: Add Non-UM Users
+---
+
+   <div id="content-wrap">
+    <h1>Add Non UM user Documentation</h1>
+	<p>Follow documentation for this Features</p>
+   </div>
```

Comment:
Done, Let me know what do you think

Reply:
Looks a bit better, but to be honest, I'm still seeing 4, 1, and 2 in different places for HTML. I think I would go with 4 or 2.

Reply:
I fixed all now

Reply:
Oh, sorry, looks like maybe you interpreted my "one indentation scheme" as meaning use one space for indentation. I meant let's only use a single indentation size (so 2 spaces all the time, rather than 2 spaces some times, 4 spaces other times).